### PR TITLE
Changes to prevent fallback formatting of empty Run key event #2912

### PR DESCRIPTION
**Pull request by joachimmetz**
Changes to prevent fallback formatting of empty Run key event #2912

### CODE REVIEW

File: plaso/parsers/winreg_plugins/run.py [hunk 1]
```diff
@@ -84,7 +84,9 @@ def ExtractEvents(self, parser_mediator, registry_key, **kwargs):
       entries.append(value_string)
 
     event_data = RunKeyEventData()
-    event_data.entries = ' '.join(sorted(entries)) or None
+    # If entries is empty make sure it is set to an empty sting otherwise
```
Comment (Onager) [inline plaso/parsers/winreg_plugins/run.py:87]: string, not sting

Comment (joachimmetz) [reply plaso/parsers/winreg_plugins/run.py:87]: Not sure what you mean can you be more specific.

**Comment by joachimmetz** [comment]
@Onager can you be a bit more verbose in your comments

File: plaso/parsers/winreg_plugins/run.py (hunk 1 above)
Comment (joachimmetz) [reply plaso/parsers/winreg_plugins/run.py:87]: per conversation you meant there was a typo in "sting"

**Comment by joachimmetz** [comment]
@Onager per conversation typo addressed

**Pull request merged**